week_exercises/w_1_common.py

```diff
@@ -0,0 +1,11 @@
+"""
+Common utility functions for week 1 exercises
+"""
+
+def test_function_output_for_input_equals_expected(function, input, output_expected,
+    should_unpack_input=False, verbose=True):
+    output_actual = function(*input) if should_unpack_input else function(input)
```

Comment:
`should_unpack_input` is not strictly needed for exercise 1 and 2 but I need it for the upcoming exercise 3